base/applications/mscutils/servman/control.c

```diff
@@ -39,21 +42,22 @@ DoControlService(LPWSTR ServiceName,
             break;
         default:
             /* Unhandled control code */
-            return FALSE;
+            DPRINT1("Unknown control command: 0x%X\n", Control);
+            return SC_MANAGER_ERR_UNKCMD;
     }
 
     hSCManager = OpenSCManagerW(NULL,
                                 NULL,
                                 SC_MANAGER_CONNECT);
-    if (!hSCManager) return FALSE;
+    if (!hSCManager) return SC_MANAGER_ERR_SRVMGROPN;
```

Comment:
Error code is set by the function call, why you returning it yourself? (and same in other places)

Reply:
Will be rewritten with use of GetLastError & co